### PR TITLE
[feature](merge-cloud) rewrite cloud table property

### PR DESCRIPTION
**Pull request by yujun777**
## Proposed changes

Issue Number: close #xxx

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by yujun777** [comment]
run buildall

**Comment by yujun777** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java [hunk 1]
```diff
@@ -194,6 +195,55 @@ public class PropertyAnalyzer {
...
+        if (properties == null) {
+            return;
+        }
+        for (String property : properties.keySet()) {
+            if (!unsupportedProperties.containsKey(property)) {
+                continue;
+            }
+
+            String value = unsupportedProperties.get(property);
+            if (value != null) {
```
Comment (luwei16) [inline fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java:236]: value == null ？

**Comment by yujun777** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java (hunk 1 above)
Comment (yujun777) [reply fe/fe-core/src/main/java/org/apache/doris/common/util/PropertyAnalyzer.java]: update

**Comment by yujun777** [comment]
run buildall

**Review by luwei16 — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**